kiva/agg/src/kiva_graphics_context.h

```diff
@@ -588,7 +588,7 @@ namespace kiva
 
 
             agg24::path_storage img_outline = img.boundary_path(img_mtx);
-            agg24::rendering_buffer* src_buf = img.rendering_buffer_ptr();
```

Comment:
That's the first thing I tried but it doesn't compile on windows. I get a template error diarrhea. Here is an excerpt.

    kiva\agg\src\kiva_graphics_context.h(711): note: while compiling class template member function 'kiva::graphics_context<agg24::pixfmt_abgr32>::graphics_context(unsigned char *,int,int,int,kiva::interpolation_e)'
    kiva\agg\agg_wrap.cpp(4177): note: see reference to function template instantiation 'kiva::graphics_context<agg24::pixfmt_abgr32>::graphics_context(unsigned char *,int,int,int,kiva::interpolation_e)' being compiled
    kiva\agg\src\kiva_graphics_context.h(618): error C2664: 'agg24::image_accessor_clip<agg24::pixfmt_rgb24>::image_accessor_clip(agg24::image_accessor_clip<agg24::pixfmt_rgb24> &&)': cannot convert argument 1 from 'agg24::pixfmt_abgr32' to 'agg24::pixfmt_alpha_blend_rgb<agg24::blender_rgb24,agg24::rendering_buffer,3,0> &'